README.md

```diff
@@ -75,7 +75,9 @@ Resources that work on Nano Server:
 * **ConfigurationPath**: Folder location where DSC configurations are stored.
 * **RegistrationKeyPath**: Folder location where DSC pull server registration key file is stored.
 * **AcceptSelfSignedCertificate**: Whether self signed certificate can be used to setup pull server.
-* **UseUpToDateSecuritySettings**: Whether to use enhanced security settings for the node where pull server resides on.
+* **UseSecurityBestPractices**: Whether to use best practice security settings for the node where pull server resides on.
+Caution: Setting this property to $true will reset registry values under "HKLM:\SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL". This environment change may break some other applications.
```

Comment:
>This environment change may break some other applications. [](start = 142, length = 58)

There's a couple of links with info about this registry key. I am not sure if they apply to the specific change we are doing, but if they do it may be helpful to include them.

Maybe something along the lines of

      "This change enforces the use of stronger encryption cypher and may affect legacy applications. More information can be found at [https://support.microsoft.com/en-us/kb/245030](https://support.microsoft.com/en-us/kb/245030) and [https://technet.microsoft.com/en-us/library/dn786418(v=ws.11).aspx](https://technet.microsoft.com/en-us/library/dn786418(v=ws.11).aspx)."

All looks good, so signing off, thanks for the updates